src/app/components/Calendar.tsx

```diff
@@ -86,9 +88,17 @@ const Calendar = ({ admin = false }) => {
         <style>{calendarStyles}</style>
         <>
         <div className={style.buttonContainer}>
-          <Button mt={3} ref={btnRef} onClick={onOpen} colorScheme="teal">
-            Add Event
-          </Button>
+          {admin ? (
+            <div style={{ display: 'flex', justifyContent: 'space-between', width: '100%' }}>
+            <Link href="/admin/profiles">
+              <Button mt={3} colorScheme="teal">
+                Profile Database
+              </Button>
+            </Link>
+            <Button mt={3} ref={btnRef} onClick={onOpen} colorScheme="teal">
+              Add Event
+            </Button>
+          </div>) : null}
```

Comment:
May want to move this to the top to save vertical space, but this is good for now.